rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -186,14 +321,60 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
+      type: object
+      required:
+        - transformationStatus
+        - metadataInformations
+      properties:
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
```

Comment:
❗️ I would add a note describing that this is an aggregated status and that unless it reports completion or failure, the request is still being processed.